lib/support/browserScript.js

```diff
@@ -126,7 +155,11 @@ async function findAndParseScripts(root, category) {
       const name = path.basename(filepath, '.js');
       const script = await readFile(filepath, 'utf8');
 
-      result[name] = generateScriptObject(name, root, script);
+      try {
+        result[name] = generateScriptObject(name, root, script);
+      } catch (e) {
+        log.error('Could not parse user script ' + name + ': ' + e);
```

Comment:
Why is this a problem now?  Can we avoid this whole business by having `generateScriptObject` do whatever it would have done before?